payments/invoice/other-features.md

```diff
@@ -19,108 +19,677 @@ sidebar:
-          "failedReason": "",
-          "isOperational": false,
-          "operations": [
-            {
-              "href": "https://api.externalintegration.payex.com/psp/invoice/payments/<payments-id>",
-              "rel": "edit-authorization",
-              "method": "PATCH"
-            }
-          ]
-        }
+        "type": "CreditCard",
+        "amount": 1500,
+        "vatAmount": 0
+      },
+      {
+        "type": "Visa",
+        "amount": 1500,
+        "vatAmount": 0
+      },
+      {
+        "type": "MasterCard",
+        "amount": 1500,
+        "vatAmount": 0
+      }
+    ],
+    "description": "Test Purchase",
+    "payerReference": "AB1234",
+    "generatePaymentToken": false,
+    "generateRecurrenceToken": false,
+    "userAgent": "Mozilla/5.0...",
+    "language": "sv-SE",
+    "urls": {
+      "hostUrls": [
+        "https://example.com"
+      ],
+      "completeUrl": "https://example.com/payment-completed",
+      "cancelUrl": "https://example.com/payment-canceled",
+      "paymentUrl": "http://example.com/perform-payment",
+      "callbackUrl": "https://example.com/payment-callback",
+      "logoUrl": "https://example.com/payment-logo.png",
+      "termsOfServiceUrl": "https://example.com/payment-terms.pdf"
+    },
+    "payeeInfo": {
+      "payeeId": "12345678-1234-1234-1234-123456789012",
+      "payeeReference": "CD1234",
+      "payeeName": "Merchant1",
+      "productCategory": "A123",
+      "orderReference": "or123",
+      "subsite": "MySubsite"
+    },
+    "metadata": {
+      "key1": "value1",
+      "key2": 2,
+      "key3": 3.1,
+      "key4": false
+    },
+    "cardholder": {
+      "firstName": "firstname/companyname",
+      "lastName": "lastname",
+      "email": "leia.ahlstrom@swedbankpay.com",
+      "msisdn": "+4673900000",
+      "homePhoneNumber": "homePhoneNumber",
+      "workPhoneNumber": "workPhoneNumber",
+      "shippingAddress": {
+        "firstName": "Leia",
+        "lastName": "Ahlstrom",
+        "email": "leia.ahlstrom@swedbankpay.com",
+        "msisdn": "+46739000001",
+        "streetAddress": "Helgestavägen 9",
+        "coAddress": "coAddress",
+        "city": "19792 Bro",
+        "zipCode": "XXXXX",
+        "countryCode": "countryCode"
+      },
+      "billingAddress": {
+        "firstName": "firstname/companyname",
+        "lastName": "lastname",
+        "email": "email",
+        "msisdn": "msisdn",
+        "streetAddress": "streetAddress",
+        "coAddress": "coAddress",
+        "city": "city",
+        "zipCode": "zipCode",
+        "countryCode": "countrycode"
+      },
+      "accountInfo": {
+        "accountAgeIndicator": "01",
+        "accountChangeIndicator": "01",
+        "accountPwdChangeIndicator": "01",
+        "shippingAddressUsageIndicator": "01",
+        "shippingNameIndicator": "01",
+        "suspiciousAccountActivity": "01",
+        "addressMatchIndicator": false
       }
-    ]
+    },
```

Comment:
Please delete everything that has to do with riskIndication (this is for 3D secure) and everything concerning CreditCard should not be in invoice